Makefile

```diff
@@ -103,6 +105,18 @@ travisci-checks: fmt-check vet-check lint                    ## Pass fmt / vet &
 travisci: archive travisci-checks travisci-cover             ## Executed by TravisCI
 .PHONY: travisci
 
+terraform-test: archive .bin/terraform                            ## Test the Terraform code
+	@.bin/terraform init terraform
+	AWS_DEFAULT_REGION=us-east-1 .bin/terraform validate -var lambda_zipname=$(LOCAL_PATH)/lambda.zip terraform/
+	AWS_DEFAULT_REGION=us-east-1 .bin/terraform validate -var lambda_s3_bucket=bucket -var lambda_s3_key=key terraform/
+.PHONY: terraform-test
```

Comment:
The indent of the comment `## Test the Terraform code` would have to be reduced, you can type `make help` to understand why :)